router/pump.go

```diff
@@ -153,67 +153,25 @@ func (p *LogsPump) pumpLogs(event *docker.APIEvents, backlog bool) {
 		debug("pump.pumpLogs():", id, "ignored: environ ignore")
 		return
 	}
-	if !logDriverSupported(container) {
```

Comment:
This drops the following functionality https://github.com/gliderlabs/logspout/commit/324db6e01dda3aa6727c03cb3f29217728bd861f.